pygama/dsp/_processors/template.py

```diff
@@ -0,0 +1,77 @@
+# 1) Import python modules
+#
+import numpy as np
+from numba import guvectorize
+
+# 2) Provide instructions to numba
+# 
+# Documentation about numba guvectorize decorator:
+# https://numba.pydata.org/numba-doc/latest/user/vectorize.html#the-guvectorize-decorator
+#
+# Notes:
+# - use `nopython=True` and `cache=True`         (@ian, is this correct?)
```

Comment:
In most cases this is correct. The exceptions are:
- if the processor relies on a command that numba does not recognize, then you want to use `objectmode=True`. For example, if you are using something not on this list of accepted numpy operations: https://numba.pydata.org/numba-doc/dev/reference/numpysupported.html, or if you are wrapping a function from another module
- `cache=True` does not apply if you are using a factory function to apply some sort of initialization to the processor

Reply:
I see. Do you think it is worth adding these comments to the template?